src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -548,30 +509,47 @@ public static int execute(final String[] args) throws Exception {
             }
 
             try {
-                if(issueWarnings(tc) != 0) {
+                if(issueWarnings(restHighLevelClient) != 0) {
                     return (-1);
                 }
             } catch (Exception e1) {
                 System.out.println("Unable to check whether cluster is sane");
                 throw e1;
             }
 
-            if(updateSettings != null) { 
+            if(updateSettings != null) {
                 Settings indexSettings = Settings.builder().put("index.number_of_replicas", updateSettings).build();
-                ConfigUpdateResponse res = tc.execute(ConfigUpdateAction.INSTANCE, new ConfigUpdateRequest(getTypes(true))).actionGet();
-                if(res.hasFailures()) {
-                    System.out.println("ERR: Unabe to reload config due to "+res.failures());
+				Response res = restHighLevelClient.getLowLevelClient()
+						.performRequest(new Request("PUT", "/_plugins/_security/configupdate?config_types=" + Joiner.on(",").join(getTypes(true))));
+
+                if (res.getStatusLine().getStatusCode() != 200) {
+					System.out.println("Unable to reload configuration because return code was " + res.getStatusLine());
+					return (-1);
                 }
-                final AcknowledgedResponse response = tc.admin().indices().updateSettings((new UpdateSettingsRequest(index).settings(indexSettings))).actionGet();
+
+				JsonNode resNode = DefaultObjectMapper.objectMapper.readTree(res.getEntity().getContent());
+
+				if (resNode.get("configupdate_response").get("has_failures").asBoolean()) {
+					System.out.println("ERR: Unable to reload config due to " + responseToString(res, false) + "/" + resNode);
+				}
+				final AcknowledgedResponse response = restHighLevelClient.indices().putSettings((new UpdateSettingsRequest(index).settings(indexSettings)), RequestOptions.DEFAULT);
                 System.out.println("Reload config on all nodes");
                 System.out.println("Update number of replicas to "+(updateSettings) +" with result: "+response.isAcknowledged());
-                return ((response.isAcknowledged() && !res.hasFailures())?0:-1);
+				return ((response.isAcknowledged() && !resNode.get("configupdate_response").get("has_failures").asBoolean()) ? 0 : -1);
             }
-            
-            if(reload) { 
-                ConfigUpdateResponse res = tc.execute(ConfigUpdateAction.INSTANCE, new ConfigUpdateRequest(getTypes(false))).actionGet();                
-                if(res.hasFailures()) {
-                    System.out.println("ERR: Unabe to reload config due to "+res.failures());
+
+            if(reload) {
+                Response res = restHighLevelClient.getLowLevelClient()
+                        .performRequest(new Request("PUT", "/_plugins/_security/configupdate?config_types=" + Joiner.on(",").join(getTypes(false))));
```

Comment:
Same question as above: why can't it use  the existing endpoint `/securityconfig`?